askama_shared/src/generator.rs

```diff
@@ -436,7 +446,7 @@ impl<'a> Generator<'a> {
 
     fn write_call(&mut self, state: &'a State, ws: &WS, name: &str, args: &[Expr]) {
         let def = state.macros.get(name).expect(&format!("macro '{}' not found", name));
-        self.handle_ws(ws);
+        self.flush_ws(ws);
```

Comment:
This didn't make sense to me at first, but I think the need for this change is explained by the preparing/flushing `ws1` and `ws2` from the macro definition.